docs/quickstart.rst

```diff
@@ -40,9 +40,46 @@ For example, to install or update the latest version using pip:
 .. _PyPI: https://pypi.org/project/Flask-SQLAlchemy/
 
 
+Initialize the Extension
+------------------------
+
+First create the ``db`` object using the ``SQLAlchemy`` constructor.
+
+.. code-block:: python
+
+    from flask import Flask
+    from flask_sqlalchemy import SQLAlchemy
+    from sqlalchemy.orm import DeclarativeBase
+
+    db = SQLAlchemy()
+
+
+By default, this extension assumes that you are using the SQLAlchemy 1.x API for defining models.
```

Comment:
I've reworded to use the first sentence, but I dislike the passive voice in the second sentence, since that makes it unclear who exactly is doing the model defining.